esp/scm/ws_config2.ecm

```diff
@@ -41,6 +41,7 @@ ESPresponse [exceptions_inline] EmptyResponse
 
 ESPresponse StatusResponse
 {
+    bool EnvironmentModified;
```

Comment:
this should be part of statustype struct, unless statustype is really an "errors" structure

Reply:
StatusType is just the status array. It is reused in other response structures.